src/esp_littlefs.c

```diff
@@ -323,8 +354,17 @@ esp_err_t esp_vfs_littlefs_register(const esp_vfs_littlefs_conf_t * conf)
     }
 
     int index;
-    if(conf->partition_label)
-    {
+
+#ifdef CONFIG_LITTLEFS_SDMMC_SUPPORT
+    if (conf->sdcard) {
```

Comment:
Instead of sticking this at the beginning of the if/else block, i'd stick it in the middle so it can be more cleanly macro'd out:

```
}
#ifdef CONFIG_LITTLEFS_SDMMC_SUPPORT
else if (conf->sdcard) {
...
}
#endif
else{ ...
```